scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -317,6 +317,18 @@ task LoadTable {
 
     DIR="~{storage_location}/~{datatype}_tsvs/"
 
+    # check for load lock
+    LOCKFILE="loadlock"
+    HAS_LOCKFILE=$(gsutil ls "${DIR}${LOCKFILE}" | wc -l)
+    if [ $HAS_LOCKFILE -gt 0 ]; then
+      echo "ERROR: load lock in place. Check whether a load is in progress or a previous load had an error."
+      exit 1
+    else  # put the lock file in place
+      echo "Setting load lock"
+      touch $LOCKFILE
```

Comment:
I came across a similar problem in my BQ locking thing... and while I think it's maybe overkill might be useful here to not touch but instead echo some random number into that file... then below (after the if) cat the contents of the file to ensure that this process is he one that wrote it.

Reply:
💥  Just die

Reply:
implemented this suggestion - we cat a uuid into the lockfile and check that it matches 👍